README.md

```diff
@@ -107,6 +107,9 @@ If you want karma to automatically rerun the tests on file modification, use:
 
     ./script/karma start
 
+### Multilingual
+Do not forget to run `rake tmp:cache:clear` after locales are updated to reload I18n js translations.
```

Comment:
@oeoeaio, Good question! I don't know. In dev environment it is not enough to restart the server. So I assume it won't be enough in production as well... Is there a way I could test it?